### PR TITLE
Update 40-stash-cache-plugin.cfg

### PR DESCRIPTION
**Pull request by ivukotic**
adding these rather important performance tuning parameters. These defaults are very sensible.

### CODE REVIEW

File: configs/stash-cache/config.d/40-stash-cache-plugin.cfg [hunk 1]
```diff
@@ -80,6 +80,20 @@ else
    set eSPACEHIGHWM=0.95
 fi
 
+if defined ?~WQ_BLOCKS_PER_LOOP
+   set eWQBLOCKS=$WQ_BLOCKS_PER_LOOP
+else
```
Comment (brianhlin) [inline configs/stash-cache/config.d/40-stash-cache-plugin.cfg:85]:
```suggestion
if defined ?~XC_WQ_BLOCKS
   set eWQBLOCKS=$XC_WQ_BLOCKS
else
```

To follow formatting of other vars

File: configs/stash-cache/config.d/40-stash-cache-plugin.cfg [hunk 2]
```diff
@@ -80,6 +80,20 @@ else
...
+   set eWQBLOCKS=10
+fi
+
+if defined ?~WQ_THREADS
+   set eWQTHREADS=$WQ_THREADS
```
Comment (brianhlin) [inline configs/stash-cache/config.d/40-stash-cache-plugin.cfg:90]:
```suggestion
if defined ?~XC_WQ_THREADS
   set eWQTHREADS=$XC_WQ_THREADS
```

**Review by brianhlin — CHANGES_REQUESTED** [review]
Some suggestions for a uniform env var naming format

**Comment by brianhlin** [comment]
@ivukotic please re-submit with an improved commit message per https://github.com/opensciencegrid/xcache/pull/102#pullrequestreview-563703022

**Comment by ivukotic** [comment]
how do I add commit comment when doing it from browser?

**Comment by brianhlin** [comment]
See step 5 here: https://docs.github.com/en/free-pro-team@latest/github/managing-files-in-a-repository/editing-files-in-your-repository

The top text box is the commit summary (what you see in `git log`) and the bottom box is if you want to add more details about the change that are too large for a short summary.

**Pull request closed without merging**